ggmolvis/renderer.py

```diff
@@ -53,9 +53,13 @@ def __init__(self, resolution=(640, 360), filepath=None):
         bpy.context.scene.render.ffmpeg.format = 'MPEG4'
         bpy.context.scene.render.ffmpeg.codec = 'H264'
         bpy.context.scene.render.ffmpeg.constant_rate_factor = 'HIGH'
+        self.frame_bounds = frame_bounds
     
     def render(self):
         # Render the movie
+        if self.frame_bounds is not None:
+            bpy.context.scene.frame_start = self.frame_bounds[0]
+            bpy.context.scene.frame_end = self.frame_bounds[1]
```

Comment:
if it is `None`, we might also consider defaulting to the trajectory bounds, similar to the MN initialization approach Brady mentioned? (since a new user may not know about the kwarg)

Reply:
I agree default / `None` should default to trajectory bounds. If a user is going to be rendering a subset / outside of that they'll have to explicit anyway.